kibbeh/src/modules/layouts/ElectronHeader.tsx

```diff
@@ -0,0 +1,54 @@
+import isElectron from "is-electron";
+import React from "react";
+import { WinMaximizeIcon, WinCloseIcon, WinMinimizeIcon } from "../../icons";
+import { Button } from "../../ui/Button";
+
+let ipcRenderer: any = undefined;
+if (isElectron()) {
+  ipcRenderer = window.require("electron").ipcRenderer;
+}
+
+export function ElectronHeader() {
+  return (
+    <div className="electron-header z-50">
+      <div className="header-image-cont">
+        <img
+          className="header-image"
+          src="https://github.com/benawad/dogehouse/raw/staging/kibbeh/public/img/doge.png"
+          width="20px"
+          height="20px"
+        />
+      </div>
+      <p className="header-title">DogeHouse</p>
+      <div className="header-icons flex flex-row-reverse w-full space-x-2 space-x-reverse">
+        <Button
+          icon={<WinCloseIcon width={10} height={10} />}
+          size="small"
+          color="secondary"
+          className="remove-outline hover:bg-accent-hover"
+          onClick={() => {
+            if (isElectron()) ipcRenderer.send("@app/quit");
+          }}
+        />
+        <Button
+          icon={<WinMaximizeIcon width={10} height={10} />}
+          size="small"
+          color="secondary"
+          className="remove-outline"
+          onClick={() => {
+            if (isElectron()) ipcRenderer.send("@app/maximize");
+          }}
+        />
+        <Button
+          icon={<WinMinimizeIcon width={10} height={10} />}
+          size="small"
+          color="secondary"
+          className="remove-outline"
+          onClick={() => {
+            if (isElectron()) ipcRenderer.send("@app/minimize");
+          }}
+        />
```

Comment:
Using the UI button is not meant for things like this. Using the normal `<button>` will get rid of the rounded edges (titlebars usually don't have them) and it will make it more dynamic and easier to change.

Reply:
> Fixing the scrollbar popping up when the header is present would also be good as it currently pops up to the side because of the header, which is not how it is supposed to act.
> ![image](https://user-images.githubusercontent.com/50546858/114724050-79fa2500-9d3b-11eb-969d-89e8fee9e3f4.png)

why should the scrollbar not be present?

Reply:
Because without the titlebar the scrollbar isn't there, so why should it be there when it's present?